src/library/baseplaylistfeature.cpp

```diff
@@ -19,9 +19,17 @@
 #include "moc_baseplaylistfeature.cpp"
 #include "track/track.h"
 #include "util/assert.h"
+#include "util/file.h"
 #include "widget/wlibrary.h"
 #include "widget/wlibrarytextbrowser.h"
 
+namespace {
+
+const ConfigKey kConfigKeyLastImportExportPlaylistDirectory(
+        "[Library]", "last_import_export_playlist_directory");
```

Comment:
This changes the key. Was this intentionally?
This breaks the feature for users after upgrade.

Reply:
The same applies to the Crate version

Reply:
I thought if I touch it I might as well apply snake_case.
"This breaks the feature" is a bit of an exxageration, users just need to select it again. But if that's not okay for a bugfix release I'll revert it, of course.

Reply:
Revert or keep it?

Reply:
this is a good question.  In general, our control object and configkey names are a gigantic mess and totally inconsistent.  In general, we have preferred to stick to existing names, even if the names were poorly chosen, to preserve compatibility.  Unless there's a compelling reason to change the name, I think it's better to stick with the old one.